tests/wiseagents/test_tools.py

```diff
@@ -83,6 +84,7 @@ def stop(self):
 #Install ollam from their official website https://ollama.com/
 #run ollama with the following command: ollama run llama3.1
 
+@pytest.mark.skip(reason="does not pass CI/CD")
```

Comment:
This tools test and test_coordinator both make use of llama 3.1 so we'll also need to set up Ollama for these tests to pass.

Reply:
Ah, I missed that since I didn't see a script for this. Will fix next week